docs/readmes/lte/upgrade_1_6.md

```diff
@@ -0,0 +1,26 @@
+---
+id: upgrade_1_6
+title: Upgrade to v1.6
+hide_title: true
+---
+
+# Upgrade to v1.6
+
+### Fresh Install Notes
+
+Instructions on fresh gateway installs are provided in the [Install AGW section](https://magma.github.io/magma/docs/lte/deploy_install)
+
+### Upgrading from previous releases
+
+Beginning with release v1.5 (Fuji), Magma artifacts are now hosted on the new Magmacore artifactory. Gateways migrating from older Magma releases should run the migration script to update the sources accordingly.
+
+To upgrade an existing AGW, please run the following upgrade script
+
+```bash
+wget https://raw.githubusercontent.com/magma/magma/v1.6/lte/gateway/release/upgrade_magma.sh
+./upgrade_magma_sh
```

Comment:
looks like wget does not preserve file permissions, can you add explicit `bash` to the command.

Reply:
> looks like wget does not preserve file permissions, can you add explicit `bash` to the command.

upgrade script should be referenced from `master` instead of `1.6` >
`wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/release/upgrade_magma.sh`

`Pull Request` 
https://github.com/magma/magma/pull/7887

`Upgrade Logs` 
https://gist.github.com/shanku9/c98812437d5af10b89b7b18296c30537